### ISSUE TITLE
"Edit tags" dialog box usability issues for games with many tags

### ISSUE THREAD

**Issue by salty-horse** [iftechfoundation/ifdb#913]
In the game page, when I click "Edit tags", it always opens the tags dialog so its top is just over the "Edit tags" button, and then it puts the focus on the text input field at the bottom of the dialog box.

When viewing a game with a lot of tags (like [Counterfeit Monkey](https://ifdb.org/viewgame?id=aearuuxv83plclpl)), this can mean that the top of the opened dialog box is off-screen, and important instructions, including the name of the dialog box, are hidden from the user.

The focus on the text input field has another side-effect. In the example game page with lots of tags, it scrolls the page down, way below the tags area. So when you confirm the box, you do not see the visual indication of "Saved" that's in the tags area, and you don't see the current state of your tags. 

It would be nice if the box would open relative to the viewport, without scrolling the page. If it's too large to fit the screen, there should be a scroll bar inside the dialog box.

**Comment by dfabulich** [comment]
If and when we ever implement tracking tags in game history #377, I'd love to move the tag editor to `editgame`, and probably redo it.

(But I'm afraid that this intuition may be tricking me into doing nothing for now, in favor of doing the "perfect thing" far, far in the future.)

**Comment by salty-horse** [comment]
LOL #914

**Comment by salty-horse** [comment]
"Edit game" is about editing the game info for everyone, no? I'm thinking about the dialog that lets users add their own tags.

**Comment by dfabulich** [comment]
Yeah, IMO the whole idea of users "owning" tags is bad/wrong. Tagging should just be shared public data about a game, same as the description, which anybody can edit, tracked in history.

**Comment by dfabulich** [comment]
I've only just now realized that I never really wrote that idea down, so I've filed https://github.com/iftechfoundation/ifdb/issues/915
